docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -13,26 +13,71 @@ To adopt the {compute_service_first_ref}, you patch an existing `OpenStackContro
 * You have completed the previous adoption steps.
 * You have defined the following shell variables. Replace the following example values with the values that are correct for your environment:
 ----
-$ alias openstack="oc exec -t openstackclient -- openstack"
+alias openstack="oc exec -t openstackclient -- openstack"
+
+DEFAULT_CELL_NAME="cell3"
+RENAMED_CELLS="cell1 cell2 $DEFAULT_CELL_NAME"
+----
++
+The `default` cell takes a new name from `DEFAULT_CELL_NAME`.
+In a multi-cell adoption scenario, it may retain its original 'default' name as well.
+
+A standalone TripleO only creates a default cell, so you should define that instead:
++
+----
+DEFAULT_CELL_NAME="cell1"
+RENAMED_CELLS="cell1"
 ----
++
 
 .Procedure
 
 . Patch the `OpenStackControlPlane` CR to deploy the {compute_service}:
 +
 [NOTE]
 This procedure assumes that {compute_service} metadata is deployed on the top level and not on each cell level. If the {OpenStackShort} deployment has a per-cell metadata deployment, adjust the following patch as needed. You cannot run the metadata service in `cell0`.
+To enable local cells metadata services, set `spec.nova.template.cellTemplates.cell*.metadataServiceTemplate.enable` in `OpenStackControlPlane` CR.
 +
 [source,yaml]
 ----
-$ oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+rm -f celltemplates
+for CELL in $(echo $RENAMED_CELLS); do
+  cat >> celltemplates << EOF
+        ${CELL}:
+          hasAPIAccess: true
```

Comment:
Does the source deployment version (17.1) supports cells without upcall support? If so then we need to detect if a source cell has no upcall support configured (i.e. the cell conductor has no api_database config) and reflect that in the destination cloud by setting hasAPIAccess: false.

This probably OK to handle as a follow up but please file a tracker for it to avoid forgetting it.